### ISSUE TITLE
Dash-table experiments not found (renderer issue)

### ISSUE THREAD

**Issue by rauschbe** [plotly/dash-table-experiments#102]
Hello,

I was going through similar problems last night and this morning but could not find a workaround for the following issue. 

While initiating and running the code there seems to be no problem. The local site unfortunately does not show any datatable and shows the following error:

` 
    Error: dash_table_experiments was not found.

    at Object.resolve (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:41843:15)

    at isLoadingComponent (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:37471:21)

    at getLoadingState (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:37508:15)

    at http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:37554:36

    at computeMergedProps (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:33212:25)

    at Connect.updateMergedPropsIfNeeded (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:33326:31)

    at Connect.render (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557404001:33450:41)

    at ce (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/react-dom@16.8.6.min.js?v=0.23.0&m=1557404001:98:136)

    at qg (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/react-dom@16.8.6.min.js?v=0.23.0&m=1557404001:97:440)

    at hi (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/react-dom@16.8.6.min.js?v=0.23.0&m=1557404001:104:285)`

I am really new in working with dash or any related packages. Any help is highly appreciated!

Many thanks!

**Comment by wsebastiangroves** [comment]
I've been having the exact same issue:

(This error originated from the built-in JavaScript code that runs Dash apps. Click to see the full stack trace or open your browser's console.)
Error: dash_table_experiments was not found.

    at Object.resolve (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:41843:15)

    at isLoadingComponent (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:37471:21)

    at getLoadingState (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:37508:15)

    at http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:37554:36

    at computeMergedProps (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:33212:25)

    at Connect.updateMergedPropsIfNeeded (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:33326:31)

    at Connect.render (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/dash_renderer.dev.js?v=0.23.0&m=1557503513:33450:41)

    at ce (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/react-dom@16.8.6.min.js?v=0.23.0&m=1557503513:98:136)

    at qg (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/react-dom@16.8.6.min.js?v=0.23.0&m=1557503513:97:440)

    at hi (http://127.0.0.1:8050/_dash-component-suites/dash_renderer/react-dom@16.8.6.min.js?v=0.23.0&m=1557503513:104:285)


Been scouring the internet for answers for 24 hours now. Yours is the first question that I could find similar to my issue. I tried updating all my dependencies to no avail.

Python initiates and runs the code with no problem (including all calls to dash_table_experiments), but when I go to my local Dash there's a blank white page with the above error listed.

**Comment by rauschbe** [comment]
Thanks for contributing! If you find a solution, I would highly appreciate it to keep me posted in some way. 
It also seems to be a problem related to the dash-renderer package. Maybe it would make sense to post the issue there as well.

**Comment by wsebastiangroves** [comment]
> Thanks for contributing! If you find a solution, I would highly appreciate it to keep me posted in some way.
> It also seems to be a problem related to the dash-renderer package. Maybe it would make sense to post the issue there as well.

I opened a thread in the renderer subforum on github:
https://github.com/plotly/dash-renderer/issues/171

I was told that dash_table_experiments is actually deprecated in favor of dash_table.

Here's the link I used to get a jump start on dash_table usage in my app:
https://dash.plot.ly/datatable/interactivity

I've since been able to render my first dash and view it in my browser! I hope that the link helps, good luck!
